pkg/kv/kvserver/replica_raft.go

```diff
@@ -1523,7 +1530,7 @@ func (m lastUpdateTimesMap) isFollowerActive(
 		// replicas were updated).
 		return false
 	}
-	return now.Sub(lastUpdateTime) <= MaxQuotaReplicaLivenessDuration
```

Comment:
Is this now unused? Looks like it to me. Some tests use it but those need fixing?

Reply:
It still had a user outside of tests. But I've added a commit to get rid of it. PTAL.